source/templates/conditionals.md

```diff
@@ -1,7 +1,49 @@
-Sometimes you may only want to display part of your template if a property
-exists.
+Conditional statements bring a minimum of logic into Ember templating.
+Statements like `if` and `unless` are implemented as built-in helpers. Helpers
+can be invoked three ways.
 
-We can use the `{{#if}}` helper to conditionally render a block:
+The first style of invocation is **inline invocation**. This looks similar to
+displaying a property, but helpers accept arguments. For example:
+
+```handlebars
+<div>
+  {{if isFast "zoooom" "putt-putt-putt"}}
+</div>
+```
+
+`{{if}}` in this case returns `"zoooom"` when `isFast` is true and
+`"putt-putt-putt"` when `isFast` is false. Helpers invoked as inline expressions
+render a single value, the same way that properties are a single value.
+
+Inline helpers don't need to be used inside HTML tags. They can also be used
+inside attribute values:
+
+```handlebars
+<div class="is-car {{if isFast "zoooom" "putt-putt-putt"}}">
+</div>
+```
+
+**Nested invocation** is another way to use a helper. Just like inline helpers,
+nested helpers generate and return a single value. For example, this template
+only renders `"zoooom"` if both `isFast` and `isFueled` are true:
+
+```handlebars
+<div>
+  {{if isFast (if isFueled "zoooom")}}
+</div>
+```
+
+The nested helper is called first returning `"zoooom"` only if `isFueled` is
+true. Then the inline expression is called, rendering the nested helper's
+value (`"zoooom"`) only if `isFast` is true.
+
+The third form of helper usage is **block invocation**. Use block helpers
+to render only part of a template. Block invocation of a helper can be
+recognized by the `#` before the helper name, and the closing `{{/` mustache
```

Comment:
I think remove the word "mustache" - it doesn't add anything, and not everybody knows to turn their head sideways to the left ;)